system/modules/core/classes/FrontendTemplate.php

```diff
@@ -251,7 +251,7 @@ protected function addToCache()
 			$strHeader = sprintf
 			(
 				"<?php /* %s */ \$expire = %d; \$content = %s; \$type = %s; \$files = %s; \$assets = %s; ?>\n",
-				$strCacheKey,
+				ltrim($strCacheKey, '*'),
```

Comment:
Just to comment on why this is necessary: when the cache key starts with `*`, the resulting PHP header of the cache file would look like this:
```php
<?php /* */empty.de */ $expire = 1494230344; $content = 'text/html'; $type = 'regular'; $files = ''; $assets = ''; ?>
```
This is invalid PHP syntax of course, thus I simply remove the `*` for the header.